packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -978,9 +978,11 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
       return null
     }
 
+    const schemaDid = response.result.data?.txn.metadata['from'] as string
```

Comment:
this also works?

```suggestion
    const schemaDid = response.result.data?.txn.metadata.from as string
```

Reply:
Also -- I think this is solved by https://github.com/openwallet-foundation/credo-ts/pull/1884

Reply:
I was planning on remove this because of the merged PR